content/intro-to-storybook/svelte/pt/get-started.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Tutorial Storybook para o Svelte'
+tocTitle: 'Get started'
+description: 'Configure o Storybook em seu ambiente de desenvolvimento'
+---
+
+Storybook executa paralelamente à aplicação em modo de desenvolvimento. Ajuda a construir componentes de interface de usuário (UI) isolados à lógica de negócio e contexto da aplicação. Esta edição do tutorial Introdução ao Storybook é para Svelte; outras edições existem para [Vue](/intro-to-storybook/vue/pt/get-started), [Angular](/intro-to-storybook/angular/pt/get-started), [React](/intro-to-storybook/react/pt/get-started), [React Native](/intro-to-storybook/react-native/pt/get-started) e [Ember](/intro-to-storybook/ember/pt/get-started).
+
+![Storybook e sua aplicação](/intro-to-storybook/storybook-relationship.jpg)
+
+## Set up Svelte Storybook
+
+Precisaremos seguir alguns passos para configurar o processo de compilação em nosso ambiente de desenvolvimento. Para começar, queremos utilizar [degit](https://github.com/Rich-Harris/degit) para configurar nosso sistema de compilação. Utilizando este pacote, é possível obter "modelos" (aplicações parcialmente construídas com configurações padrão) para ajudar a acelerar o fluxo de desenvolvimento da aplicação.
+
+Execute os seguintes comandos:
+
+```shell:clipboard=false
+# Criar nossa aplicação:
+npx degit chromaui/intro-storybook-svelte-template taskbox
+
+cd taskbox
+
+# Instalar dependências
+yarn
+```
+
+<div class="aside">
+💡 Este modelo contém estilos, recursos e configurações essenciais, todos necessários para esta versão do tutorial.
+</div>
+
+Agora é possível verificar que os diversos ambientes de nossa aplicação estão funcionando corretamente:
+
+```shell:clipboard=false
+# Executar o testador (Jest) em linha de comando:
```

Comment:
@yamgomes hope you don't mind, but could you keep the snippet comments in English? I know it brings the translation "closer to home" (pardon the bad pun). But it's our guideline to leave them as such. Sorry for not being more explicit with those in the Readme. I completely forgot about it.